src/hooks/useToken.js

```diff
@@ -0,0 +1,24 @@
+import { useEffect } from 'react';
+
+function useToken() {
+  let myHeaders = new Headers();
+  myHeaders.append('Content-Type', 'application/json');
+
+  let raw = JSON.stringify({
+    user: 'igorioha95',
+    password: 'Va1mXa610o7v'
```

Comment:
I shouldn't be able to see your credentials. You also shouldn't store 'secret' information on the client.

Reply:
You mean, that I need to store credentials in another file or store them in local storage when I make autorisation?